app/views/subscription_mailer/notify_tag_added.html.erb

```diff
@@ -0,0 +1,19 @@
+<p>Dear <%= @user.username %>,</p>
+
+<p>Public Lab contributor <a href='https://<%= ActionMailer::Base.default_url_options[:host] %>/profile/<%= @node.author.name %>'><%= @node.author.name %></a> just added a new tag
```

Comment:
So here, we need the name of the person who added the tag, not the person who posted the note. They may not be the same!